Sources/Fuzzilli/Core/JavaScriptEnvironment.swift

```diff
@@ -195,20 +195,27 @@ public class JavaScriptEnvironment: ComponentBase, Environment {
         logger.info("Have \(customPropertyNames.count) custom property names: \(customPropertyNames)")
         logger.info("Have \(customMethodNames.count) custom method names: \(customMethodNames)")
     }
-    
+
+    public var constructables: [String] {
+        // These are basically the groups that we already filter for in the
+        // init function and some that can not be constructed with the new
+        // keyword
+        groups.keys.filter({ !$0.contains("Constructor") && !["Math", "JSON", "Reflect", "Symbol"].contains($0) })
```

Comment:
Can you compute this list at the end of the initialize function? It doesn't have to be computed on every access since it won't change afterwards, or can it?